znoyder/config.yml

```diff
@@ -286,6 +286,44 @@ add:
         pipeline:
           - 'check'
 
+  'nova':
+    'osp-16.1':
+      'osp-rpm-functional-py36':
+        pipeline:
+          - 'check'
+        vars:
+          extra_commands:
+            - dnf install -y python3-ddt python3-ironicclient python3-mock python3-oslotest python3-osprofiler python3-placement-tests python3-stestr python3-testresources python3-testscenarios python3-wsgi_intercept
+      'osp-rpm-py36':
```

Comment:
You missed this bit from the code.eng patch [1]:

```
rhos_release_args: '16.2'
rhos_release_extra_repos: rhelosp-16.2-unittest
nodeset: pod-rhel-8.4
```

The jobs don't work without those, as we need the newer content from 16.2 as opposed to 16.2-cdn,  the correct nodeset, and the extra dependencies from -unittest repo.

[1] https://code.engineering.redhat.com/gerrit/c/openstack/osp-internal-jobs/+/439278/23/zuul.d/cre-nova.yaml#14

Reply:
+1, did not notice that. @sdatko Could you provide an example of the generated job by znoyder? Just to compare with what we did manually.

Reply:
Just for transparency, the `rhos_release_*` things and `nodeset` were set for all jobs to the same **per-release**, so OSP 16.2 uses one set of values, OSP 17 uses another.
The issue was that for some reason, I am not sure why, the `16.2-cdn` repos were picked instead of just `16.2` for jobs. I changed it in next pull request https://github.com/RedHatCRE/znoyder/pull/125 and this is resolved now.